src/firefly/plans/xafs_scan.py

```diff
@@ -34,6 +35,15 @@ def k_step_to_E_step_round(k_start, k_step):
     return round(k_step_to_E_step(k_start, k_step), 2)
 
 
+def time_converter(total_seconds):
```

Comment:
This looks like a duplicate of ``firefly.plans.line_scan.LineScanDisplay.time_converter``. Can we have one implementation of this and use it in both places?

Reply:
Maybe put this function in some common util python files?

Reply:
Yeah, that sounds right. Feel free to add a utils.py file in Firefly if it makes sense.

Reply:
added util.py in plans folder